test.js

```diff
@@ -52,6 +52,18 @@ test('toVFile()', function (t) {
     st.equal(first, second)
     st.end()
   })
+
+  t.test('should accept a WHATWG URL object', function (st) {
+    var file = toVFile(new URL('file:///foo/bar/baz.qux'))
```

Comment:
Looks like windows is failing. Perhaps better to not hardcode this, but get it by combining a path with `import.meta.url`?